listenbrainz/db/playlist.py

```diff
@@ -67,6 +67,7 @@ def get_by_mbid(playlist_id: str, load_recordings: bool = True) -> Optional[mode
             user = db_user.get(user_id)
             if user:
                 collaborators.append(user["musicbrainz_id"])
+            collaborators.sort()
```

Comment:
I think these two lines should be outside the for loop. And given that this code is duplicated 3 times, we should make a helper function and call that 3 times.

Reply:
4 times actually !